unison-src/transcripts-round-trip/main.output.md

```diff
@@ -820,18 +820,43 @@ sloppyDocEval =
 These are currently all expected to have different hashes on round trip.
 
 ```ucm
-.> diff.namespace a3 a3_old
+scratch/main> diff.namespace /a3_new:. /a3:.
 
   Updates:
   
-    1. sloppyDocEval : Doc2
+    1. sloppyDocEval : #ej86si0ur1
        ↓
-    2. sloppyDocEval : Doc2
+    2. sloppyDocEval : #ej86si0ur1
 
 ```
 ## Other regression tests not covered by above
 
-### Comment out builtins in the edit command
+### Builtins should appear commended out in the edit command
 
 Regression test for https://github.com/unisonweb/unison/pull/3548
 
+```ucm
```

Comment:
I changed it to actually show that the builtins added to the scratchfile are commented out.